deploy/helm/sumologic/values.yaml

```diff
@@ -181,6 +187,15 @@ sumologic:
     ## ref: https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter#configuration
     bearerCacheTtl: "3600"
 
+
+## Configure metrics-server
+## ref: https://github.com/helm/charts/blob/master/stable/metrics-server/values.yaml
+metrics-server:
```

Comment:
do we need to teach our CI to generate `metrics-server-overrides.yaml` for the non-Helm installation?

Reply:
I guess it will be required if we want to add the arguments.